Docs/source/usage/parameters.rst

```diff
@@ -1502,6 +1502,10 @@ Particle initialization
     Resampling is performed everytime the number of macroparticles per cell of the species
     averaged over the whole simulation domain exceeds this parameter.
 
+* ``<species>.do_temperature_deposition`` (`boolean`) optional (default `false`)
+    When running with Ohm's Law Hybrid Solver, this will enable temperature deposition
+    in each dimension with a matched shape function and filtering used for current deposition.
+    This is required when using the electron energy solver with electron-ion temperature relaxation.
```

Comment:
When the electron energy solver is being used, is `do_temperature_deposition` turned on automatically? I don't see that here.

Reply:
We are still working to debug and test that PR internally. I can add a reference to it when we submit the PR to the WarpX main repo. This would be in preparation for the electron energy solver. I figured I would forget to update this comment later.